proxy/envoy/config.go

```diff
@@ -651,7 +664,7 @@ func buildEgressFromSidecarVirtualHostOnPort(rule *proxyconfig.EgressRule, mesh
 	externalTrafficRoute := buildDefaultRoute(externalTrafficCluster)
 
 	return &VirtualHost{
-		Name:    rule.Name + "-" + strconv.Itoa(port.Port),
+		Name:    rule.Domains[0] + "-" + strconv.Itoa(port.Port),
```

Comment:
Would 2 rules with same domains[0] create problems ?

Reply:
No, domains are unique across the config.